src/Common/ResetMon.cpp

```diff
@@ -36,7 +37,11 @@
 
 #include <Logging.h>
 
+#if defined(CONFIG_IDF_TARGET_ESP32S3)
+#include "esp32s3/rom/crc.h"
+#else
 #include "esp32/rom/crc.h"
+#endif

```

Comment:
Remove the include, because already included in the header.

Reply:
nope, header inclued rom/**rtc.**.h not **crc**.h

Reply:
Oups, please include on lower level to avoid preprocessor ifdef

Reply:
removed esp32/ pfad part in #include